src/components/FeedbackPrompt.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { Button, Form, Modal, Message } from 'semantic-ui-react';
+import 'isomorphic-fetch';
+
+// URL to direct requests to, from the Google Apps Script
+const postUrl = 'https://script.google.com/macros/s/AKfycbyXYbemTPcqsdbmXITnjaNi-CkN85g5kKPrgzt4AS8ykT2jH6Zn/exec';
+
+/**
+* Maps short question names to input type and question text
+*/
+const questions = {
+  currentSnap: {
+    inputType: Form.Input,
+    text: 'If amount for the CURRENT SNAP subsidy was wrong, what\'s the correct amount?'
+  },
+  futureSnap: {
+    inputType: Form.Input,
+    text: 'If amount for the FUTURE SNAP subsidy was wrong, what\'s the correct amount?'
+  },
+  futureS8: {
+    inputType: Form.Input,
+    text: 'If amount for the FUTURE Section 8 voucher was wrong, what\'s the correct amount?'
+  },
+  otherCircumstances: {
+    inputType: Form.TextArea,
+    text: 'What else could be going on that could affect your benefit amount? ' +
+      'For example, are you a veteran? Are you a full-time student?'
+  },
+  bugReport: {
+    inputType: Form.TextArea,
+    text: 'If there was a bug or error, describe the bug and what you were trying to do when the bug happened.'
+  },
+  comments: {
+    inputType: Form.TextArea,
+    text: 'Do you have any other comments?'
+  }
+};
+
+/**
+* Form for submitting user feedback.
+*/
+class FeedbackForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      submissionFailed: false,
+    };
```

Comment:
This state isn't used.

Reply:
Actually it is used on line 144.

Reply:
That is `<FeedbackPrompt>` state, not `<FeedbackForm>` state, isn't it?

Reply:
Oh you're right, sorry, that's leftover from before `<FeedbackPrompt>` existed.